extensions/ql-vscode/src/run-queries.ts

```diff
@@ -478,6 +478,10 @@ export async function compileAndRunQueryAgainstDatabase(
   // Figure out the library path for the query.
   const packConfig = await cliServer.resolveLibraryPath(diskWorkspaceFolders, queryPath);
 
+  if (!packConfig.dbscheme) {
+    throw new Error('No dbscheme. Is your qlpack.yml missing or invalid?');
```

Comment:
Only we use the term dbscheme, so this can be explained a bit more.
```suggestion
    throw new Error('Could not find a database scheme for this query. Please check that you have a valid qlpack.yml file for this query, which refers to a database scheme either in the `dbscheme` field or through one of its dependencies.');
```